test/cql-pytest/api_util.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+
+# This file provides utility to REST API requests.
+# Some metrics cannot be obtained by nodetool, but they are available by API.
+
+import requests
+import nodetool
+import pytest
+
+# Sends GET request to REST API. Response is restured as JSON.
+# If API isn't available, `pytest.skip()` is called.
+def get_request(cql, *path):
+    if nodetool.has_rest_api(cql):
+        response = requests.get(f"{nodetool.rest_api_url(cql)}/{'/'.join(path)}")
+        return response.json()
+    else:
+        pytest.skip("REST API not available")
+
+# Get column family's metric which requires column family name parameter in
+# keyspace:name format.
+def get_column_family_name_metric(cql, metric, table):
+    ks, cf = table.split('.')
+    return get_request(cql, "column_family", "metrics",
+      "bloom_filter_false_positives", f"{ks}:{cf}")
```

Comment:
You wanted to wanted to pass the given metric parameter here, but accidentally passed the constant "bloom_filter_false_positives".